packages/ffe-message-box-react/src/BaseMessage.js

```diff
@@ -21,7 +26,7 @@ const BaseMessage = props => {
                     `ffe-message-box__icon--${type}`,
                 )}
             >
-                {icon}
+                {React.cloneElement(icon, { style: iconStyles, ...icon.props })}
```

Comment:
Could you add the possibility of overriding these styles in part?

```js
React.cloneElement(icon, { ...icon.props, style: { ...iconStyles, ...icon.props.style } });
```js

Reply:
Oh, that. I was asking about opinions on that in the issue but didn't get any so I decided to just keep it simple.